nilearn_sandbox/datasets.py

```diff
@@ -0,0 +1,70 @@
+import os
+import cookielib
+import getpass
+
+from sklearn.datasets.base import Bunch
+
+from nilearn import datasets
+from nilearn._utils.compat import _urllib
+
+from ._utils.compat import _input
+
+
+def _get_nitrc_auth_cookie(login=None):
+    if login is None:
+        login = _input('login: ')
+    passwd = getpass.getpass('password: ')
```

Comment:
I prefer to pass username / password as parameters, and input the data within example scripts (from environment variables, or from command-line parameters).